WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/PostsCardViewModel.swift

```diff
@@ -30,7 +30,7 @@ class PostsCardViewModel: NSObject {
 
     private let postService: PostService
 
-    private var postListFilter: PostListFilter = PostListFilter.draftFilter()
+    private var postListFilter: PostListFilter = PostListFilter.draftFilter(includeRecentlyTrashed: false)
```

Comment:
I guess it probably would be good to keep the posts list and postcard in sync in terms of how they work, this is probably going to make things simple.